src/asciidoc/docs/platform-features/richPreview.adoc

```diff
@@ -1,143 +1,61 @@
-For optimal rich preview support, design this view with the following considerations in mind.
+To customize the thumbnail for your app, render a custom view when the URL contains the `Koji.playerState.context=screenshot` query parameter.
+For optimal thumbnail support, design this view with the following considerations in mind.
 
 * The image is responsive.
 * The image looks good with a square aspect ratio.
 * The image is optimized for faster loading.
 For example, to reduce the image size to 400x400 pixels and load only the first frame of a gif, you can add query parameters at the end of your image's URL: `\https://images.koji-cdn.com/path/to/myimage.gif?fit=bounds&width=400&height=400&frame=1`
-* The image includes customizable elements to generate unique previews of each remix.
+* The image includes customizable elements to generate unique previews for each customized version of the app.
 
-image::Koji-screenshot=1.svg[Render a view for the rich preview image]
+image::Koji-screenshot=1.png[Render a view for the thumbnail image]
 
-For the following example, assume the template supports remixes of the title, font size, and logo.
+For the following example, assume the app supports customizing the image defined by `customImage`.
 
-. In your application code, add a conditional statement to display a different image for social shares.
+. In your application code, add a conditional statement to display a custom image for social shares.
 For example:
 +
-[.tabs,scope="lang"]
---
-.React
-====
-[.intro]
-Add a conditional statement in the `render` function.
-
-[source,JavaScript]
-----
-render() {
-  if (window.location.search.includes('koji-screenshot=1')) {
-    return (
-      <SocialSharePreview>
-          <H1
-          fontSize={this.state.titleOptions.fontSize}
-          >{this.state.titleOptions.title}</H1>
-          <Image src={this.optimizeURL(this.state.logo)} />
-        </SocialSharePreview>
-    );
-  }
-  ...
-}
-----
-====
-
-.Vanilla JS
-====
-[.intro]
-Wrap a conditional statement around the `render` function.
-
-[source,JavaScript]
-----
-// render the social share preview
-const socialShareRender = () => {
-    document.body.innerHTML = `
-        <div id="socialSharePreview">
-            <h1 style="font-size:${titleSize}px" id="title">${title}</h1>
-            <img id="logo" src="${optimizeURL(logo)}"/>
-        </div>`;
-}
-
-// render
-if (window.location.search.includes('koji-screenshot=1')) {
-    socialShareRender();
-} else {
-    render();
-}
-----
-====
-
-.Angular
-====
-[.intro]
-Add a conditional statement in the `ngOnInit` function and use it to apply a dynamic `id`.
-
-[.tabs]
-=====
-.app.component.ts
 [source,JavaScript]
 ----
-ngOnInit() {
-  ...
-  this.social = false
-  if (window.location.search.includes('koji-screenshot=1')) {
-    this.social = true
-  }
-  ...
+...
+if (Koji.playerState.context === 'screenshot') {
+  return (
+    <SocialSharePreview>
+      <h1>This is a custom thumbnail.</h1>
```

Comment:
It might be better to use a custom value here (you get the custom values in remix mode, you use them in other modes, like default and screenshot).
```suggestion
      <h1>{remixValues.title}</h1>
```